pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -166,9 +89,9 @@
                 key={ caption }
                 className="apply"
                 variant={ variant }
-                isLoading={ this.state.action_in_progress && this.state.action_caption_in_progress == caption }
+                isLoading={ this.props.action_in_progress && this.props.action_caption_in_progress == caption }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19200-20230821-124816-2fdc0a03-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)